examples/common/python/connectors/direct/tcs_listener/tcs_listener.py

```diff
@@ -257,10 +259,15 @@ def parse_command_line(config, args):
         config['Logging']['LogLevel'] = options.loglevel.upper()
     if options.bind_uri:
         bind_uri = options.bind_uri
+    if options.lmdb_url:
```

Comment:
bind_uri is listener uri where will listening i/p requests and lmdb_url is lmdb server url. I don't why we choose only port for avalon listener and host is hard coded to localhost. We can change it in another PR.

Reply:
I think we need to maintain a list of changes that will be addressed in separate PR. since there are too many changes suggested which are going to be handled in separate PR. Else it's difficult to track. Could you raise issue in github ?